test/cxx/netcdf_test_fixture.cc

```
@@ -0,0 +1,37 @@
+// NetCDFTestFixture.cpp
+#include "netcdf_test_fixture.h"
+
+void NetCDFTestFixture::SetUp() {
```

cppcheck (2.10-2)

[style] The function 'SetUp' is never used.

(unusedFunction)